.gitpod.yml

```diff
@@ -21,4 +21,4 @@ vscode:
   extensions:
     - dbaeumer.vscode-eslint
     - esbenp.prettier-vscode
-    - juanblanco.solidity
\ No newline at end of file
+    - juanblanco.solidity
```

Comment:
this looks like a VSCode extension for solidity, but this isn't a solidity project. I know this is just for your gitpod but is it necessary?

Reply:
Good catch. It was there when I started. I don't think we need it since front end doesn't touch Solidity code. Even for e2e tests, we should only need binaries. Gone.